designer/server/src/main/scala/pl/touk/nussknacker/ui/server/AkkaHttpBasedRouteProvider.scala

```diff
@@ -857,6 +877,36 @@ class AkkaHttpBasedRouteProvider(
     additionalUIConfigProviderFactory.create(config, sttpBackend)
   }
 
+  private def createCustomHttpServiceProvider(
+      config: Config,
+      processService: ProcessService
+  ): Resource[IO, Map[String, CustomHttpServiceProvider]] = {
+    Multiplicity(
```

Comment:
maybe we can move the SPI loading-related code to some well-named private method?